pep-0458-tuf-online-keys.rst

```diff
@@ -571,19 +900,19 @@ Project developers expect the distributions they upload to PyPI to be
 immediately available for download.  Unfortunately, there will be problems when
 many readers and writers simultaneously access the same metadata and
 distributions.  That is, there needs to be a way to ensure consistency of
-metadata and repository files when multiple developers simulaneously change the
-same metadata or distributions.  There are also issues with consistency on PyPI
-without TUF, but the problem is more severe with signed metadata that MUST keep
-track of the files available on PyPI in real-time.
+metadata and repository files when multiple developers simultaneously update
+the same metadata or distributions.  Without TUF, there are also issues with
+consistency on PyPI, but the problem is more severe with signed metadata that
+MUST keep track in real-time the files available on PyPI.
```

Comment:
906: revise "keep track in real-time the files" to "keep track, in real-time, of the files"